gen.py

```diff
@@ -60,6 +63,27 @@ class PlotFlags(OutFlags):
     template: FilePath = Field(None, description="Default plot template")
 
 
+class TopLevelPlotFlags(BaseModel):
+    x: PlotColumn = Field(
+        None, description="Default field name to use as x-axis data"
+    )
+    y: Union[PlotColumns, Dict[FilePath, PlotColumns]] = Field(
+        default_factory=dict,
+        description=(
+            "A single column name, list of columns,"
+            " or a dictionary of data-source and column pair"
+        ),
+    )
+    x_label: str = Field(None, description="Default label for the x-axis")
+    y_label: str = Field(None, description="Default label for the y-axis")
+    title: str = Field(None, description="Default plot title")
+    template: FilePath = Field(None, description="Default plot template")
```

Comment:
Template can also be a string name of the template, but I guess since the `FilePath` is str it is ok for now? Or should we create a separate type for it?

Reply:
Well it can be a string, for example, if user writes `confusion` we will search for the template that has `DEFAULT_NAME` field with this value. If it is not found, we will start checkif if it wasn't a path.

Reply:
Okay, thanks. Is that also the case for `template` in `plots` output?